### DISCUSSION TITLE
Selection Counter does not decrease after purge and cleanup

### DISCUSSION THREAD

**Discussion by kerkhc** [photoprism/photoprism#4215] [Q&A] [answered]
Hi all,
I'm actually testing a big bunch of new photos/videos and therefore, I need to remove uploaded objects, if meta data is not as expected.
Process example (.MP4):
1. copy MP4-file and related json-file (Google TakeOut format) to original folder
2. index data using CLI
`docker compose -f /opt/photoprism-docker/docker-compose.yml exec photoprism photoprism index "[folder to be indexed]"`
3. check data in UI (via sub menu "Folder")
4. Remove MP4 and json from original folder
5. Remove sidecar files from appropriate sidecar folder
6. purge data using CLI
`docker compose -f /opt/photoprism-docker/docker-compose.yml exec photoprism photoprism purge`
7. cleanup data using CLI
`docker compose -f /opt/photoprism-docker/docker-compose.yml exec photoprism photoprism cleanup`
8. Check UI: Thumbnail is no longer available
9. Check filesystem: original folder and sidecar folder do not contain MP4 and json file
10. Check photo data in database
`docker exec -it photoprism-docker-mariadb-1 bash -l`
10.1 run sql client and select from photos table: record does no longer exist

Issue:
1. Navigate to "Folder" from sub menu "Folder"
2. UI selector does not decrease counter
3. after each new run (remove objects and upload again with corrected json), the counter increases

Image shows "17" with only one video selected although only 1 video was uploaded and removed multiple times.

This behavior seems not to block any functionality, but opening the properties (Pencil) shows 16 empty objects. The last one (no. 17) shows the expected properties.

![image](https://github.com/photoprism/photoprism/assets/42431686/e81c8f9b-b7ce-4d48-ac44-9bdd91395237)

**Comment by kerkhc** [comment]
This strongly seems to be browser-related.
After I cleared cache, cookies, etc. from the browser, the counter came up with "1" as I selected my new object.

**Reply by graciousgrey** [reply]
It might be cached in this scenario. 
Note that you can also delete files using the web ui this will remove sidecar files as well. 

https://docs.photoprism.app/user-guide/organize/delete/ 

If you remove files directly from the original folder, I would recommend re-indexing the folder so that the index can be updated accordingly.

**Reply by lastzero** [reply, marked as answer]
Existing selections are stored in the browser memory, even if you reload the page. Open the menu and click X (clear) to clear the selection. Reindexing will not do this.

**Reply by kerkhc** [reply]
sooo easy - thank you
